src/PrestaShopBundle/Resources/views/Admin/Sell/CustomerService/OrderReturn/Blocks/form.html.twig

```diff
@@ -0,0 +1,99 @@
+{#**
+ * 2007-2020 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2020 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+{{ form_start(orderReturnForm, {action: path('admin_order_returns_update', {'orderReturnId': editableOrderReturn.orderReturnId})}) }}
+<div class="card">
+  <h3 class="card-header">
+    <i class="material-icons">person</i>
+    {{ 'Merchandise returns'|trans({}, 'Admin.Orderscustomers.Feature') }}
+  </h3>
+
+  <div class="card-block row">
+    <div class="card-text">
+      {{ form_errors(orderReturnForm) }}
+
+      <div class="form-group row">
+        <label class="form-control-label">
+          {{ 'Customer'|trans({}, 'Admin.Global') }}
+        </label>
+
+        <div class="col-sm form-control-value">
+          <p>{{ editableOrderReturn.customerFullName }}</p>
+
+          <a href="{{ path('admin_customers_view', {'customerId': editableOrderReturn.customerId }) }}" class="btn btn-outline-secondary">
+            {{ 'View details on the customer page.'|trans({}, 'Admin.Orderscustomers.Help') }}
```

Comment:
@JevgenijVisockij, I can see in the screenshot that this migration has turned the former legacy hyperlink into a CTA. Do you confirm? If yes, I guess it should be reworded without punctuation and shortened like `View customer details` in `Admin.Actions`.

Reply:
Yes it did.

Alright I will change the naming, thanks!